XamlAnimatedGif/UriLoader.cs

```diff
@@ -108,11 +108,11 @@ private static Task<Stream> CreateTempFileStreamAsync(string fileName)
             return Task.FromResult(stream);
         }
 
-        private static Task DeleteTempFileAsync(string fileName)
+        private static void DeleteTempFile(string fileName)
```

Comment:
Good thing making it synchronous! It was async for historical reasons (when the library supported several platforms, including some that had only async IO methods).

Reply:
Thanks for the explanation :)